packages/protocol/contracts/identity/test/TestRandom.sol

```diff
@@ -0,0 +1,13 @@
+pragma solidity ^0.5.3;
+
+import "../Random.sol";
+
+contract TestRandom is Random {
+  function testRandomness(uint bn, bytes32 randomness) external {
```

Comment:
nit: It might be more readable for this to be `addRandomness` and have the internal function as `_addRandomness`?

Or just rename this to `addTestRandomness`?